### ISSUE TITLE
[Accessibility] There is not enough contrast on the light blue link text on the light gray background.   

### ISSUE THREAD

**Issue by dkanev** [vmware/clarity#2585]
Our product which uses clarity has VPAT assessment report which contains issue that the used color light blue (#007CBB) on gray background (#FAFAFA) is not enough contrast.

[Issue]
There is not enough contrast on the light blue link text on the light gray background. 
 
Computed Size: 12px
Foreground:#007CBB
Background:#FAFAFA
 
The contrast ratio is: 4.4:1
 
Text failed at Level AA
Text failed at Level AAA
Large text passed at Level AA
Large text failed at Level AAA
 
[Impact]
Ensure there is sufficient contrast between foreground and background colors. This requirement applies to text over solid and image backgrounds as well as images that contain text. Proper foreground and background contrast is necessary for users with low vision and some users with color blindness to accurately read content.
 
 
[Recommendation] Developers must ensure that the color contrast for all text (and images of text) meets the WCAG 2.0 Level AA requirements, unless specifically excluded. Designers or developers should modify the foreground color, background color, and/or font size so that sufficient contrast is attained. Standard text less than 18 point (or less than 14 point if bold) must have a luminosity contrast ratio of 4.5:1 or more. Text 18 point or larger (14 point or larger if bold) must have a luminosity contrast ratio of 3:1 or more. Use the Level Access contrast checking tool (https://www.levelaccess.com/color-contrast-checker/) to determine if contrast is sufficient.
 

## Use case

We use the light blue (#007CBB) for buttons, links app level alerts.
Could you suggest with which color to be change the light blue (#007CBB) or the background color in order to meet the required contrast ratio of 4.5:1.
It’s good the clarity color palate  and examples  to be updated with the new color recommendations.

The purpose solution from @mathisscott  is update that specific blue to #0079B8 to get proper contrast

Thanks

**Comment by mathisscott** [comment]
This is one we will need to backport to 0.12 and 0.11
